freespace2/freespace.cpp

```diff
@@ -1596,8 +1596,8 @@ void game_init()
 
 	Game_current_mission_filename[0] = 0;
 
-	// seed the random number generator
-	Random::seed(static_cast<unsigned int>(time(nullptr)));
+	// seed the random number generator, using a static seed if benchmarking to encourage consistency
+	Random::seed(Cmdline_benchmark_mode ? 2335 : static_cast<unsigned int>(time(nullptr)));
```

Comment:
I agree with @z64555, this should be in its own PR since it has nothing to do with the scripting changes.

Reply:
I've removed this. In all honesty it wasn't even producing more consistent benchmarks.